packages/core/src/components/DataTable.story.tsx

```diff
@@ -1,17 +1,24 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
+// @ts-ignore
```

Comment:
What is this for?

Reply:
I'm getting what I thought was just a storybook bug:

```
Could not find a declaration file for module '@airbnb/lunar-icons/lib/interface/IconStar'. '/Users/kyle_schiller/lunar/packages/icons/lib/interface/IconStar.js' implicitly has an 'any' type.
  Try `npm install @types/airbnb__lunar-icons` if it exists or add a new declaration (.d.ts) file containing `declare module '@airbnb/lunar-icons/lib/interface/IconStar';`ts(7016)
```